client/src/components/TrackerColumn.jsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+
+const TrackerColumn = ({ category }) => {
+  return (
+    <Box width="25%" mx={1}>
```

Comment:
This works, but maybe there's a better way to achieve the proper width across all the columns using flex attributes. It doesn't make a difference now, but if we ever decide to add/remove columns, it would automatically resize.